metrics/prometheus.go

```diff
@@ -119,9 +119,13 @@ func NewPrometheusCollector(i infoProvider, f ContainerLabelsFunc, includedMetri
 				help:      "Last time a container was seen by the exporter",
 				valueType: prometheus.GaugeValue,
 				getValues: func(s *info.ContainerStats) metricValues {
+					if now == nil {
+						t := time.Now()
```

Comment:
please use k8s.io/util/clock for testing time-based things.

Reply:
Very nice tool it is.